src/sass/app_2/_common/base/themes.scss

```diff
@@ -29,6 +29,8 @@ $themes: (
         text_color                       : $COLOR_BLACK,
         text_primary_color               : $COLOR_LIGHT_BLACK_1,
         text_secondary_color             : $COLOR_LIGHT_BLACK_2,
+        wip_btn_color                    : $COLOR_WHITE,
```

Comment:
we can use `themed('container_color')` for the transition from `$COLOR_WHITE` to `$COLOR_DARK_BLUE_2`

Reply:
@cakasuma , same comment as below.